assets/video_looper.ini

```diff
@@ -118,6 +118,21 @@ copyloader = false
 password = videopi
 
 
+[playlist]
+# This setting allows for a fixed playlist.
+
+# Path to the playlist file.
+# If you enter a relative path (e.g. not starting with /) it is considered relative to the selected file_reader path (directory or USB drive).
+# Leave empty to get all files in the file_reader path (directory or USB drive).
+path = 
+#path = playlist.m3u
+
+# Playlist format.
+# Auto is based on the extension.
+format = auto
```

Comment:
is this even necessary? we could just agree (and document) that file extenions for the playlist file should be m3u or m3u8?

Reply:
I admit. Well at first I wanted to implement loading PLS format as well, because it's based on INI and so it's easy to load... until I realized VLC cannot generate them. So, to do later :) Still, someone could fork and easily add a custom JSON loader for their project.

Reply:
I think the playlist format should be derived from the file extension and an additional option just adds confusion and complexity.

Reply:
I removed this imho redundant option in this pull request against your branch.
https://github.com/KoltesDigital/pi_video_looper/pull/1